app/components/password_confirmation_component.scss

```diff
@@ -0,0 +1,3 @@
+lg-password-confirmation {
```

Comment:
Default `display` of custom elements is `inline`, which can cause some unexpected layout if not otherwise accounted for. This was previously fine in the account creation flow because sibling elements were block elements so it appeared standalone, but usage in the preview page was broken without.

Before|After
---|---
![image](https://github.com/18F/identity-idp/assets/1779930/222ce23a-b23a-4c65-a789-a28945edb15b)|![image](https://github.com/18F/identity-idp/assets/1779930/5ed5fba9-e551-418e-8e0c-513429d502c7)